ext/cl_khr_command_buffer_mutable_dispatch.asciidoc

```diff
@@ -233,6 +237,7 @@ CL_DEVICE_MUTABLE_DISPATCH_CAPABILITIES_KHR        0x12B0
 
 // Property to cl_ndrange_kernel_command_properties_khr
 CL_MUTABLE_DISPATCH_UPDATABLE_FIELDS_KHR           0x12B1
+CL_MUTABLE_DISPATCH_PROMISES_KHR                   0x12B7
```

Comment:
This needs added to the XML file too, https://github.com/KhronosGroup/OpenCL-Docs/blob/c37e00f1423379a882c173c12ed1aa5ca9d8e51d/xml/cl.xml#L1782

Reply:
line 1786